1.0/docs/devguide/events.md

```diff
@@ -165,8 +158,8 @@ interactions automatically when a declarative listener is added for the event
 type.  These events fire consistently on both touch and mouse environments,
 so we recommend using these events instead of their mouse- or
 touch-specific event counterparts. This provides better interoperability with both touch and
-mouse devices.  For example, `tap` should be used instead of
-`click` for the most reliable cross-platform results.
+mouse devices.  For example, `on-tap` should be used instead of
```

Comment:
We should just use `tap` and `click` here, I think--those are the proper event names.